services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -111,10 +111,10 @@ const Asset = ({ libraryData }) => {
                   <dt className={dashboardStyles.label}>Library</dt>
                   <dd className={dashboardStyles.labelLarge}>{libraryData.content.name}</dd>
                 </dl>
-                <Link href={libraryPath}>
-                  <a className={clsx(dashboardStyles.metaLink, dashboardStyles.metaLinkLarge)}>
+                <Link href={libraryPath} passHref>
+                  <CarbonLink className={clsx(dashboardStyles.metaLinkLarge)}>
```

Comment:
```suggestion
                  <CarbonLink className={dashboardStyles.metaLinkLarge}>
```

Now that we are only using one class name here.